VISION.md

```diff
@@ -19,22 +19,22 @@ In this document we present the project:
 
 | Vision | Definition and description of vision     |
 | :---       | :---                                    |
-| *CNCF* | OpenEBS is an open source project operating within CNCF Storage Landscape guidelines, and is focused on achieving ```Graduated Level``` status.   |
-| *Persona* | Multiple storage services covering the following operating modes; ```Node Local```, ```Fabric Attached```, ```Replicated```, ```Non-Replicated```, ```Protected``` and ```non-protected```, ```Highly Available``` storage.  |
+| *CNCF* | OpenEBS is an open source project operating within CNCF Storage Landscape guidelines and is focused on achieving ```Graduated Level``` status.   |
+| *Persona* | Multiple storage services covering the following operating modes; ```Node Local```, ```Fabric Attached```, ```Replicated```, ```Non-Replicated```, ```Protected``` and ```Non-Protected```, ```Highly Available``` storage.  |
 | *Fast* | Optimized multiple storage drivers and storage media types, with a particular focus on Solid State Flash media.  |
-| *Resilient* | Storage is highly resilient and self-healing. Reads and writes are protected even as containers or nodes are terminated. Volumes and Data Writes can be replicated across nodes via replication. Snapshots and cloning provide additional levels of tolerance,  resilience & self-healing. |
+| *Resilient* | Storage is highly resilient and self-healing. Reads and writes are protected even as containers or nodes are terminated. Volumes and Data Writes can be replicated across nodes via replication. Snapshots and cloning provide additional levels of tolerance, resilience & self-healing. |
 | *Available* | Storage attached to one node can be made addressable to applications running on any node within the cluster through a ```“Nexus” Virtual SAN Fabric``` (i.e. Block mode Storage Area Network). |
-| *Native* | Implemented as ```container native storage```, a Kubernetes natively integrated component, optimized for storage contained inside a Kubernetes cluster (Hyper converged). |
-| *Enterprise grade* | A fully featured, community proven, rigorously tested, trustworthy, Mission Critical product. |
-| *Kubernetes* | OpenEBS focus is for Kubernetes on Linux. |
+| *Native* | Implemented as ```container native storage```, a Kubernetes natively integrated component, optimized for storage contained inside a Kubernetes cluster (Hyper-converged). |
+| *Enterprise grade* | A fully featured, community-proven, rigorously tested, trustworthy, Mission Critical product. |
+| *Kubernetes* | OpenEBS focus is on Kubernetes on Linux. |
```

Comment:
should be "for Kubernetes on Linux" not "on"

Reply:
I suggest "OpenEBS focus is on Kubernetes-on-Linux"

Reply:
Can I get a middle ground on this? How about the below.

`OpenEBS focus is for Kubernetes-on-Linux`